src/components/atoms/Ranges/DefaultRange/Range.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import { Input, Label, StyledWrapper } from './Range.styled';
+import { IProps } from './Range.model';
+
+const Range = ({ className, type }: IProps): JSX.Element => {
+  const [rangeValue, setRangeValue] = useState(1);
```

Comment:
type should be declared in useState

```suggestion
  const [rangeValue, setRangeValue] = useState<number>(1);
```

Reply:
ok